editor/js/report_data.js

```diff
@@ -448,15 +416,51 @@ class ReportDataWrapper {
         strategyBuilder.add();
       };
 
+      const makeForEnergyUnits = (strategyBuilder) => {
+        strategyBuilder.setStrategy((x) => self.getEnergyConsumption(x));
+
+        const normalizeEnergyUnits = (units) => {
```

Comment:
Does this happen elsewhere? If so, maybe we can do this replace in a private method and resuse.